src/common/services/api/cart.service.spec.js

```diff
@@ -115,23 +115,32 @@ describe('cart service', () => {
       transformedCartResponse.rateTotals[0].cost.display = '$51.00'
       transformedCartResponse.rateTotals[0].cost['display-with-fees'] = '$52.23'
       // Based on 8.1 JSON response added the following codes
-      transformedCartResponse.lineItems = transformedCartResponse.lineItems.map(item =>{
+      transformedCartResponse.lineItems = transformedCartResponse.lineItems.map((item, index) =>{
         item.itemfields['RECURRING-DAY-OF-MONTH'] = item.itemfields['recurring-day-of-month']
         item.itemfields['RECURRING-START-MONTH'] = item.itemfields['recurring-start-month']
         item.rate = {...item.rate, cost : [{display : 10}]}
-        item.item = {...item.item, _offer : [{_code : [{code: '5541091'}]}]}
+        if(index === 0){
+          item.item = {...item.item, _offer : [{_code : [{code: '0354433'}]}]} 
+        }
+        if(index === 1){
+          item.item = {...item.item, _offer : [{_code : [{code: '0617368'}]}]} 
+        }
+        if(index === 2){
+          item.item = {...item.item, _offer : [{_code : [{code: '5541091'}]}]} 
+        }
         return {...item, configuration:item.itemfields}
       })
     })
 
     it('should get cart, parse response, and show most recent items first', () => {
       const data = self.cartService.handleCartResponse(transformedCartResponse, '2016-10-01')
       // verify response
+      console.log(data.items);
```

Comment:
Let's remove this

Reply:
Removed console for a printing data item in cart.service.spec.js file